ravenpy/models/rv.py

```diff
@@ -796,6 +789,30 @@ def txt_hrus(self):
         return "\n".join(txt)
 
 
+class RVP(RV):
+    def __init__(self, channel_profiles, **kwargs):
+        self._channel_profiles = channel_profiles
+        super().__init__(**kwargs)
+
+    @property
+    def channel_profiles(self):
```

Comment:
Note that in the rvp files for the emulators, there is a lot of stuff beyond the channel profile. Here, `RVP.render_to_rv` only renders a small part of the RVP. What I mean is that this is the renderer for the channel profile only, not the full RVP. 

Elsewhere in the rv code, I've sometimes used properties to actually print to template, that is, the `render_to_rv` is built in the property `getter`. For example , `RVI.supress_output` directly outputs the string to be inserted into the template. I don't think that was such a good idea, because the RV attributes are now split in two: those that return a Python object that is converted to a string by the templating, and those that return directly a string. I think this was a bad design decision done for a quick fix. Another design pattern I've used is in RVC with the `txt_basin_state`. There, you have both the python objects (hru_state, basin_state), and the `txt_hru_sate`, `txt_basin_state` doing the rendering. I think this could work here: 
- `channel_profiles` is a list or dictionary of channel profile objects
- `txt_channel_profiles` does the `to_rv` rendering. 

I'm sure we could improve on that in the future.

Reply:
Yes it makes sense, and that is exactly the pattern that I have used for the `RVH` class in my previous PR.